client/coral-embed-stream/src/graphql/utils.js

```diff
@@ -150,6 +150,19 @@ const ascending = (a, b) => {
   return 0;
 };
 
+const descending = (a, b) => ascending(a, b) * -1;
+
+export function insertSorted(nodes, comment, sortOrder = 'CHRONOLOGICAL') {
```

Comment:
My bad, I think this should be called `insertCommentSorted` as there are many things we can insert into the query state :-)